source/extensions/filters/common/expr/context.h

```diff
@@ -80,7 +80,11 @@ template <class T> class HeadersWrapper : public google::api::expr::runtime::Cel
     if (value_ == nullptr || !key.IsString()) {
       return {};
     }
-    auto out = value_->get(Http::LowerCaseString(std::string(key.StringOrDie().value())));
+    auto str = std::string(key.StringOrDie().value());
+    if (!Http::validHeaderString(str)) {
```

Comment:
Added the comment. I thought it would be too messy (doesn't fit on one line), especially since the string needs to be saved and passed to `validHeaderString()`. Is there a way to do that cleanly?

Reply:
Yeah, it definitely seems so. Thanks good with me!